examples/panorama_standalone/variables.tf

```diff
@@ -23,16 +23,31 @@ variable "networks" {
     Example of variable deployment :
 
     ```
-    vpcs = {
+    networks = {
```

Comment:
In AWS and Azurerm we named variable related to the virtual network on its service name, ie: in AzureRM it is vnets,  in AWS it is vpcs. I highly encourage you to change name of this variable to be familiar in all stacks :)

Reply:
I agree but there are a lot of names that point to it being network. Two of the most important:
- The terraform resource is called `google_compute_network`
- Within Google - you have `VPC networks`

So while in AWS it is indeed called `aws_vpc` - for google - they just stuck to `network` while VPC contains more components besides the network itself (VPC serverless access, shared VPC, etc.).

I would in google's case - the service name itself is `network` or `vpc_network` worst case. Does this reasoning seem ok with you ?

Reply:
It's fine for me than :)